jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/ExplorerToolWindow.kt

```diff
@@ -92,6 +92,7 @@ class ExplorerToolWindow(private val project: Project) : SimpleToolWindowPanel(t
     private fun createTree(model: DefaultTreeModel): Tree {
         val awsTree = Tree()
         TreeUIHelper.getInstance().installTreeSpeedSearch(awsTree)
+        @Suppress("DEPRECATION") // TODO: Remove when we drop < 192
```

Comment:
Can we tie these all to a single searchable string (e.g. `DROP_WHEN_MIN_IS_192`) and then also add that string to our `<idea-version>` / `intellijJVersions` ?

Reply:
Done, and backfilled